bucket.go

```diff
@@ -59,8 +64,8 @@ type KVStore interface {
 	Touch(k string, exp uint32) (cas uint64, err error)
 	Add(k string, exp uint32, v interface{}) (added bool, err error)
 	AddRaw(k string, exp uint32, v []byte) (added bool, err error)
-	Set(k string, exp uint32, v interface{}) error
-	SetRaw(k string, exp uint32, v []byte) error
+	Set(k string, exp uint32, upsertOptions UpsertOptions, v interface{}) error
```

Comment:
Suggest making options a pointer so that you can populate a set of default values.